packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue

```diff
@@ -222,9 +224,18 @@
       window.addEventListener('click', this.handleWindowClick);
       window.addEventListener('keydown', this.handlePopoverByKeyboard, true);
     },
+    beforeUpdate() {
+      // Essential for title updates after data finishes loading
+      this.breakpointLimit = this.title && this.title.length >= 20 ? 4 : 3;
```

Comment:
Is the title length we are checking against arbitrary?

Reply:
Ahh I see, thanks. For purposes of clarity, I think it would be great to add a comment( or use descriptive variable or both? whichever makes more sense) describing that the value `20` is what is currently specified as the maximum title length on small windows. Otherwise, this could end up being a potential source of bugs, considering that any change here would also require a change in the `AppBar.js` where this value is specified.

https://github.com/learningequality/kolibri/blob/develop/packages/kolibri/components/pages/AppBarPage/internal/AppBar.vue#L26